docs/upgrade/phpstan-level-1.md

```diff
@@ -0,0 +1,63 @@
+# Upgrade Instructions for Upgrading PHPStan to Level 1
+
+This article describes the upgrade instructions for [#1040 Upgrading PHPStan](https://github.com/shopsys/shopsys/pull/1040).
```

Comment:
link leads to "phpstan lvl 4" and that is confusing... remove the link maybe? Or if it is practical, please explain why the PR is lvl4, but you recommend only lvl1 here

Reply:
Added explanation